installfrompip.adoc

```diff
@@ -97,7 +97,7 @@ pip install --editable grr/config/grr-response-server
 pip install --no-cache-dir -f https://storage.googleapis.com/releases.grr-response.com/index.html grr-response-templates
 pip install --editable grr/config/grr-response-test
 ----
-Then edit /etc/default/grr-server and set:
+Then edit /grr/debian/grr-server.default and set:
```

Comment:
this path is set by the deb package. If you're just installing from pip you don't need to mess with this at all (but you need to do your own systemd setup, or run the worker, ui etc. manually. If you want to have all the systemd stuff set up for you by the package, then install a new version from source, you need to change this to update the systemd scripts.

So I don't think this needs changing but we need to clarify that somehow.